driver-cql-shaded/src/main/java/io/nosqlbench/activitytype/cql/statements/core/CQLStatementDefParser.java

```diff
@@ -122,6 +110,7 @@ public ParseResult(String stmt, String name, Map<String,String> bindings, Set<St
             this.missingGenerators = missingGenerators;
             this.missingAnchors = missingAnchors;
             this.statement = stmt;
+            this.bindings = bindings;
```

Comment:
since before it was always null, this probably has no impact in `getBindings()`

Reply:
This is (should be) subsumed in the more general parser classes for statements as well.